spec/models/prisoner_availability_validation_spec.rb

```diff
@@ -3,11 +3,11 @@
 
 RSpec.describe PrisonerAvailabilityValidation, type: :model do
   subject do
-    described_class.new(offender: offender,
+    described_class.new(prisoner: prisoner,
                         requested_slots: requested_slots)
   end
 
-  let(:offender) { Nomis::Offender.new(id: '123', noms_id: 'some_prisoner_number') }
+  let(:prisoner) { Nomis::Prisoner.new(id: '1234567', noms_id: 'ABC1234') }
```

Comment:
@TobyRet - Really like the fact that you've put in data that represents the actual format; makes it easier to understand the difference between these two similarly named attributes.

Reply:
👍